games-sports/speed-dreams/additional-files/speed_dreams.rdef

```diff
@@ -0,0 +1,87 @@
+resource app_flags B_SINGLE_LAUNCH | B_ARGV_ONLY;
+
+resource app_version {
+		major = 2,
```

Comment:
you could avoid hardcoding the version here.
Please check https://github.com/haikuports/haikuports/blob/master/app-backup/luckybackup/luckybackup-0.5.0.recipe#L85 for an example

Reply:
Thanks for the example. I'm looking into possibly using CMake to generate the rdef file.

Reply:
Done, thanks.